code/modules/unit_tests/unit_test.dm

```diff
@@ -28,31 +32,59 @@ GLOBAL_VAR(test_log)
 	/// The type of turf to allocate for the testing zone
 	var/test_turf_type = /turf/open/floor/plasteel
 
+	/// The type of turf IF ANY to have the outer testing zone set to
+	var/border_test_turf_type = null
+
+	/// How many turfs the reserved area is wide
+	var/reservation_width = 5
+	/// How many turfs the reserved area is high
+	var/reservation_height = 5
+	/// Whether to force gravity in the testing area
+	var/gravity = FALSE
```

Comment:
This should probably default to TRUE, I'm not sure if anything needs gravity off right now except for maybe chain_pull_through_space.